runhouse/resources/hardware/cluster.py

```diff
@@ -602,7 +608,8 @@ def _sync_image_to_cluster(self):
         use the HTTP client.
         """
         env_vars = {}
-        log_level = os.getenv("RH_LOG_LEVEL")
+        # In case the local env var RH_LOG_LEVEL is None, we set the RH_LOG_LEVEL on the cluster to be the DEFAULT_LOG_LEVEL
+        log_level = os.getenv("RH_LOG_LEVEL") or DEFAULT_LOG_LEVEL
```

Comment:
@carolineechen any objections / thoughts?

Reply:
this looks fine